deeplake/api/dataset.py

```diff
@@ -361,6 +363,7 @@ def empty(
         lock_enabled: Optional[bool] = True,
         lock_timeout: Optional[int] = 0,
         verbose: bool = True,
+        username: str = "public",
```

Comment:
Why do we have `username` parameter when it is inferred from credentials?

Reply:
Does `token` counts as creds?